index.html

```diff
@@ -55,11 +50,8 @@ <h1 class="intro">I’m Emmanuel Hagan Glad to see you!</h1>
 
       <!---End Social Link-->
     </ul>
-
-
   </section>
 
-
       <!-- Work section -->
     <section class="work">
```

Comment:
- [ ] Kindly add styling to your section container for your woks section, I can find it also ensure it has `display:grid` in it.